cli/azd/cmd/show.go

```diff
@@ -165,7 +165,8 @@ func showTypeFromLanguage(language project.ServiceLanguageKind) contracts.ShowTy
 // does not include the project file, we attempt to determine it by looking for a single .csproj/.vbproj/.fsproj file
 // in that directory. If there are multiple, an error is returned.
 func getFullPathToProjectForService(svc *project.ServiceConfig) (string, error) {
-	if svc.Language != "dotnet" {
+	language := svc.Language
```

Comment:
take a look at `showTypeFromLanguage()` from this same file.  You can call it as `showTypeFromLanguage(svc.Language)` and it will return `contracts.ShowTypeDotNet` for all the known dotnet languages (so you don't need to use the string constants below).

Once you use that function, you can change this to:

```
if contracts.ShowTypeDotNet != showTypeFromLanguage(svc.Language) {
  return svc.Path(), nil
}
```
```